spec/requests/api/v2/providers/courses/update_degree_subject_requirements_spec.rb

```diff
@@ -47,13 +47,13 @@ def perform_request(updated_degree_subject_requirements)
       expect {
         perform_request(updated_degree_subject_requirements)
       }.to change { course.reload.degree_subject_requirements }
-            .from("Must have a Maths A level.").to("Must have a Physics A level.")
+            .from("Must have a maths A level.").to("Must have a Physics A level.")
```

Comment:
Suggest changing this to: 'Must have an A level in maths', 'Must have an A level in physics'.